### PR TITLE
Begin updating links to use links.classicpress.net

### PR DESCRIPTION
**Pull request by mattyrob**
## Description
Issue #501 recommends pushing all ClassicPress.net URLs via the `link` subdomain. This is an initial PR covering three simple links to ensure the approach is correct before I scale up the changes.

## Motivation and context
Initial progress towards issue #501

## How has this been tested?
The links have been amended 

## Types of changes
New feature / enhancement

### CODE REVIEW

File: src/wp-admin/includes/class-wp-automatic-updater.php [hunk 1]
```diff
@@ -837,7 +837,7 @@ protected function send_debug_email() {
 
 If you think these failures might be due to a bug in ClassicPress, could you report it?
  * If you need general support: https://docs.classicpress.net/faq-support/
- * Or, if you're comfortable writing a bug report: https://docs.classicpress.net/testing-classicpress/#reporting-bugs
+ * Or, if you're comfortable writing a bug report: https://link.classicpress.net/testing-classicpress/#reporting-bugs
```
Comment (nylen) [inline src/wp-admin/includes/class-wp-automatic-updater.php]: As noted on the other PR, the `#hash-fragments` need to go away too. This one could probably be https://link.classicpress.net/reporting-bugs ...

File: src/wp-admin/setup-config.php [hunk 2]
```diff
@@ -176,7 +176,7 @@
 		echo '		<td><input name="prefix" id="prefix" type="text" value="cp_" size="25" /> ' .
 			sprintf(
 				'<a href="%s" target="_blank" rel="noopener">' . __( 'Learn More' ) . '</a>',
-				esc_url('https://docs.classicpress.net/installing-classicpress/#installation-steps')
+				esc_url('https://link.classicpress.net/installing-classicpress/#installation-steps')
```
Comment (nylen) [inline src/wp-admin/setup-config.php]: ... and this one could be https://link.classicpress.net/installing-classicpress/installation-steps I guess, leaving room for other `installing-classicpress/*` links in the future.

**Comment by nylen** [comment]
See latest comments on https://github.com/ClassicPress/subdomain-link/pull/1.

**Comment by viktorix** [comment]
Since we're using `gettext()` in v2.0, we can close this PR.

**Pull request closed without merging**